content/en/docs/reference/glossary/auto-mtls.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Auto mTLS
+test: n/a
+---
+
+Istio tracks the server workloads migrated to Istio proxies, and configures client proxies to send mutual TLS traffic to those workloads automatically, and to send plain text traffic to workloads without sidecars.
```

Comment:
Auto mTLS is a feature of Istio to automatically enable Mutual TLS Authentication (link to glossary entry for "Mutual TLS Authentication") on connections where both the client and server are able to handle Mutual TLS traffic, and to downgrade to plaintext traffic when either the client or server is not.

Reply:
Reworded a bit. Though I still keep the configure client to send mTLS since that's more accurate. Authentication is concept on server side, which is not the intention of the feature.